components/BillReadMore/BillReadMore.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react"
+import { Button, Modal } from 'react-bootstrap'
+
+const BillReadMore = ({ bill }) => {
+  const [showAddComment, setShowAddComment] = useState(false);
+
+  const handleShowAddComment = () => setShowAddComment(true);
+  const handleCloseAddComment = () => setShowAddComment(false);
```

Comment:
let's rename these since this no longer deals with adding comments. I think we can just remove "AddComment" from each of these, so we'd have show, setShow, handleShow, and handleClose